styles.css

```diff
@@ -8,12 +8,15 @@ body {
   width: 100vw;
   height: 100vh;
   display: flex;
-  justify-content: center;
-  align-items: center;
 }
 
 #canvas1 {
   width: 800px;
   height: 500px;
   border: 4px solid black;
+  margin: auto;

```

Comment:
If you only have 1 single child element inside a flex container, setting margin: auto on the flex item will center it automatically.